qa/rpc-tests/bipdersig-p2p.py

```diff
@@ -102,7 +102,7 @@ def get_tests(self):
             self.last_block_time += 1
             self.tip = block.sha256
             height += 1
-        yield TestInstance(test_blocks, sync_every_block=False)
+        yield TestInstance(test_blocks, sync_every_block=True)
```

Comment:
This test passed in 33 seconds on Core `master` on my box, and took more than 1/2 hour with this slow sync on BU `dev` .

A bit of a slowdown might be acceptable, but I'm guessing this goes beyond that.

Then we need to fix the test framework to take advantage of PV , because we cannot slow down tests like this because of PV... it's just not practical.